pkg/ui/tab.go

```diff
@@ -100,8 +100,13 @@ func RemoveSlide(s int, tui *TUI) {
 }
 
 // Adds a slide to the end of currently present slides
-func AddSlide(tui *TUI) {
-	tabSlide := *NewTab("bash", os.Getenv("SHELL"), tui)
+func AddSlide(tui *TUI, name string) {
+	var tabSlide TerminalTab
+	if name == "bash" {
+		tabSlide = *NewTab(name, os.Getenv("SHELL"), tui)
+	} else if name == "ocm-container" {
+		tabSlide = *NewTab(name, "ocm-container", tui)
```

Comment:
The names `bash` and `ocm-container` can be turned into constants.

Reply:
Yes sure, changing these into constants.